csp_billing_adapter/adapter.py

```diff
@@ -62,9 +62,10 @@ def main():
         log.setLevel(logging.INFO)
 
         config = Config.load_from_file(
-            os.path.expanduser('~/.config/csp-billing-adapter.yaml'),
+            os.path.expanduser('~/.config/csp_billing_adapter/config.yaml'),
```

Comment:
I would suggest hat we move the config to `/etc` . The config is expected to be delivered via an RPM package and creating an rpm package that puts stuff in `/root` will trigger all kinds of warnings during package build. Also if the adapter runs in a container we might want to set things up such that there are no users in the container at all. If the adapter runs as a daemon on a system we do not want to interfere with the space the root user may use for their own setup.

Reply:
fixed in https://github.com/SUSE-Enceladus/csp-billing-adapter/pull/38